### PR TITLE
[翻译完毕] utilities

### PR DESCRIPTION
**Pull request by isdanni**
Issue: #38

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 1]
```diff
@@ -119,47 +138,52 @@ msgid ""
...
+"        }\n"
+"    }\n"
+"}\n"
+"\n"
+"// TI_IO_DEF_VIRT();"
 
 #: ../../utilities.rst:85
 msgid "Progress Notification"
-msgstr ""
+msgstr "进程通知"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
msgstr "进展通知"
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 2]
```diff
@@ -3,27 +3,28 @@
...
 "Generated-By: Babel 2.8.0\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: zh_CN\n"
+"X-Generator: Poedit 2.3.1\n"
 
 #: ../../utilities.rst:2
 msgid "Utilities"
-msgstr ""
+msgstr "工具"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
msgstr "实用工具"
```

File: locales/zh_CN/LC_MESSAGES/utilities.po [hunk 3]
```diff
@@ -119,47 +138,52 @@ msgid ""
...
-msgstr ""
+msgstr "进程通知"
 
 #: ../../utilities.rst:87
 msgid ""
 "The taichi messager can send an email to ``$TI_MONITOR_EMAIL`` when the "
 "task finished or crashed. To enable:"
 msgstr ""
+"当任务完成或崩溃时，taichi消息传递程序可以将邮件发送到 ``"
+"$TI_MONITOR_EMAIL`` 。 启用："
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/utilities.po]:
```suggestion
"$TI_MONITOR_EMAIL`` 。 要启用的话："
```

**Review by archibate — COMMENTED** [review]
Cool! Btw, `utilities.rst` has been recently updated in taichi-dev/taichi#959, would you mind to run `make merge` and translate that updated part too? Thanks!

**Comment by isdanni** [comment]
> Cool! Btw, `utilities.rst` has been recently updated in [taichi-dev/taichi#959](https://github.com/taichi-dev/taichi/pull/959), would you mind to run `make merge` and translate that updated part too? Thanks!

Sure! Will update once finished ;)

**Comment by archibate** [comment]
Thank for the efficient and hard work! merging..

**Comment by archibate** [comment]
> Sure! Will update once finished ;)

Feel free to add another PR to update this!

**Pull request merged**